### PR TITLE
feat: adding Add functionality for buttons to redirect to Verification or account creation in case of POI POA verified

### PR DESCRIPTION
**Pull request by hamza-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/stores/src/mockStore.ts [hunk 1]
```diff
@@ -294,6 +300,8 @@ const mock = (): TStores & { is_mock: boolean } => {
             setSelectedAccount: jest.fn(),
             toggleAccountTransferModal: jest.fn(),
             selectAccountType: jest.fn(),
+            // getAccount: jest.fn(),
+            // toggleAccountTypeModalVisibility: jest.fn(),
```
Comment (shontzu-deriv) [inline packages/stores/src/mockStore.ts]: same here 👆🏼

File: packages/stores/types.ts [hunk 2]
```diff
@@ -315,6 +320,7 @@ type TUiStore = {
     populateHeaderExtensions: (header_items: JSX.Element | null) => void;
     populateSettingsExtensions: (menu_items: Array<TPopulateSettingsExtensionsMenuItem> | null) => void;
     setShouldShowCooldownModal: (value: boolean) => void;
+    // openSwitchToRealAccountModal: () => void;
```
Comment (shontzu-deriv) [inline packages/stores/types.ts]: 👆🏼

File: packages/stores/types.ts [hunk 3]
```diff
@@ -384,6 +390,8 @@ type TTradersHubStore = {
     is_demo: boolean;
     selectAccountType: (account_type: string) => void;
     available_cfd_accounts: TAvailableCFDAccounts[];
+    // getAccount: () => void;
+    // toggleAccountTypeModalVisibility: () => void;
```
Comment (shontzu-deriv) [inline packages/stores/types.ts]: 👆🏼

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx [hunk 4]
```diff
@@ -69,6 +71,7 @@ const CompareCFDs = observer(() => {
                         </div>
                     </div>
                 </div>
+                {/* <CFDPasswordModal context={store} platform={platform} /> */}
```
Comment (shontzu-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx]: was this comment intentional?

Comment (hamza-deriv) [reply packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx]: I was checking something.
Will remove it

File: packages/cfd/src/Components/props.types.ts [hunk 5]
```diff
@@ -269,3 +274,21 @@ export type TAvailableCFDAccounts = {
 export type TCompareAccountsCard = {
     trading_platforms: TModifiedTradingPlatformAvailableAccount;
 };
+
+export type TJurisdictionData = Record<
+    'jurisdiction',
+    'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest' | 'malta' | 'seychelles' | undefined
```
Comment (hirad-deriv) [inline packages/cfd/src/Components/props.types.ts]: We used to have 'seychelles' but not anymore. I have not seen this jurisdiction in our platforms before.

Comment (hamza-deriv) [reply packages/cfd/src/Components/props.types.ts]: just copied from previous type.
Will remove it

**Review by hirad-deriv — APPROVED** [review]

**Pull request merged**